eventfiles/class_tests/event_file_tests.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from eventfiles.models import EventFile
+from users.tests import user0  # noqa: F401
+from django.core.files import File
+from django.core.exceptions import ValidationError
+from events.tests import event_participant_not_creator, new_event  # noqa: F401
+
+TEST_FILE_PATH = 'static/test_files/testFile.txt'
+FILE_ALREADY_EXISTS = 'that file already exist in meeting'
+
+
+@pytest.fixture()
+def file0():
+    return File(open(TEST_FILE_PATH))
+
+
+@pytest.fixture()
+def event_file0(file0, event_participant_not_creator):  # noqa: F811
+    return EventFile(file=file0, participant_id=event_participant_not_creator)
+
+
+@pytest.fixture()
+def persist_event_file(event_file0):
+    event_file0.participant_id.event_id.save()
+    event_file0.participant_id.user_id.save()
+    event_file0.participant_id.save()
+    event_file0.save()
+    return event_file0
```

Comment:
In my opinion you can use conftest.py file as i mentioned in this PR:
https://github.com/redhat-beyond/lets-meet/pull/72#discussion_r861576738, for the eventfile app and put there all the constants and fixtures. 
I think that it can make our code more readable and maintainable 😄

Here is link to conftest.py documentation:
https://docs.pytest.org/en/6.2.x/fixture.html#conftest-py-sharing-fixtures-across-multiple-files

Reply:
I think it would be better to make all conftest changes in our project in a different PR, so I opened an issue for that https://github.com/redhat-beyond/lets-meet/issues/81 😄

Reply:
No problem, will be handled

Reply:
NP, will be handled 😄